src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -310,7 +310,7 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.on('desktop_set_global_mouse_selection', (event, selection: string) => {
-      clipboard.writeText(selection, 'clipboard');
+      clipboard.writeText(selection, process.platform === 'linux' ? 'selection' : 'clipboard');
```

Comment:
```suggestion
      clipboard.writeText(selection, 'selection');
```

It would be an error to try and call this routine on a non-Linux platform (since "global mouse selection" is a Linux-only concept) so I think we should just always use 'selection' here.

Reply:
Usages on the GWT side are typically guarded via `isLinuxDesktop()`; e.g.

https://github.com/rstudio/rstudio/blob/9ab60a7b67570e0b0497b76afb8daab0b291421b/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorMixins.java#L32-L67

https://github.com/rstudio/rstudio/blob/9ab60a7b67570e0b0497b76afb8daab0b291421b/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorMixins.java#L92-L124

(It's also possible that these Ace mixins are no longer necessary with Electron though?)

Reply:
Usages on the GWT side are typically guarded via `isLinuxDesktop()`; e.g.

https://github.com/rstudio/rstudio/blob/9ab60a7b67570e0b0497b76afb8daab0b291421b/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorMixins.java#L32-L67

https://github.com/rstudio/rstudio/blob/9ab60a7b67570e0b0497b76afb8daab0b291421b/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/AceEditorMixins.java#L92-L124

(It's also possible that these Ace mixins are no longer necessary with Electron though?)

Reply:
Thanks for the info! Fixed on [5628c28](https://github.com/rstudio/rstudio/pull/11431/commits/5628c28723d1cda56af907343fdfa05bdce8fbdf)